Marlin/Marlin_main.cpp

```diff
@@ -782,7 +782,7 @@ extern "C" {
 #endif // !SDSUPPORT
 
 #if ENABLED(DIGIPOT_I2C)
-  extern void digipot_i2c_set_current(int channel, float current);
+  extern void digipot_i2c_set_current(uint8_t channel, float current);
```

Comment:
I just checked and with this change:  https://github.com/fiveangle/Marlin/commit/326102424a38c6e20e6b379a2d7b3446e0a9ab3f

I get:

Sketch uses 847**48** bytes (65%) of program storage space. Maximum is 130048 bytes.
Global variables use 32**63** bytes (39%) of dynamic memory, leaving 4929 bytes for local variables. Maximum is 8192 bytes.


Sketch uses 847**42** bytes (65%) of program storage space. Maximum is 130048 bytes.
Global variables use 32**59** bytes (39%) of dynamic memory, leaving 4933 bytes for local variables. Maximum is 8192 bytes.

Is Global variables the SRAM ?  If not, how do you guys check that ?